### ISSUE TITLE
Kwikset does have an official API

### ISSUE THREAD

**Issue by Sperryfreak01** [explosivo22/kwikset-ha#83]
Kwikset does have an officially supported API ,  shoot me a message and lets see if we can make something happen.

**Comment by explosivo22** [comment]
This using the seam API?

**Comment by Sperryfreak01** [comment]
Nope, Seam was doing their own thing and then they switched to our official API.

**Comment by Sperryfreak01** [comment]
I added you to a private repo with my contact info

**Comment by Smashburgers** [comment]
Interested.

**Comment by pconwell** [comment]
Is the official Kwikset API documented anywhere? I cannot find anything - I assume it's private / invite only?

**Comment by explosivo22** [comment]
I have reached out and heard back. They do have an official API and there is a lot more to it that they are working through. I think things look promising, but there are going to be a few things that have to be worked out on the backend before anything can start happening. Once things start happening, I will be sure to keep everyone updated.

**Comment by namannik** [comment]
@explosivo22, @Sperryfreak01 -- Is there any new information about the official API?  I'm willing to help if there's anything I can do.

**Comment by Sperryfreak01** [comment]
It's all on our side, we need to do some work to support individual user integrations vs an aggregated integration.  I can't comment on timeline unfortunately but we are looking into it.

**Comment by namannik** [comment]
Ok, thanks. If there’s anything I can do to help test or do development on, let me know.
